engine/src/main/scala/cromwell/engine/workflow/OldStyleWorkflowActor.scala

```diff
@@ -768,7 +768,7 @@ case class OldStyleWorkflowActor(workflow: OldStyleWorkflowDescriptor)
 
   private def persistStatus(storeKey: ExecutionStoreKey,
                             executionStatus: ExecutionStatus,
-                            callOutputs: Option[CallOutputs] = None,
+                            callOutputs: Option[JobOutputs] = None,
```

Comment:
The variable names here and below on 797 are still `callOutputs` instead of the renamed `jobOutputs`.